### ISSUE TITLE
Character sometimes doesn't move after lighting a log

### ISSUE THREAD

**Issue by Paleocene** [2011Scape/tek5-game#41]
Followup issue after #40 is merged

## Current Behavior
After lighting a log, the character sometimes doesn't move.

## Expected Behavior
The character should move to the first available tile in the directions west, east, south, north.

## Steps to Reproduce the Problem
  1. Move to the spot in the screenshot
  2. Light a fire
  3. The character doesn't move

The cause seems to be that the two unblocked directions (east and south) both return true on `world.collision::isClipped`. See the `Player.findWesternTile()` function. That shouldn't matter for moving after lighting a fire though.

![image](https://user-images.githubusercontent.com/123772937/220122216-fbf7f0d8-f6a6-4654-a739-ef1906946379.png)
